mne/decoding/time_gen.py

```diff
@@ -98,15 +95,6 @@ class GeneralizationAcrossTime(object):
             Duration of each classifier (in seconds). By default, equals one
             time sample.
         If None, empty dict. Defaults to None.
-    predict_type : {'predict', 'predict_proba', 'decision_function'}
-        Indicates the type of prediction:
-            'predict' : generates a categorical estimate of each trial.
-
-            'predict_proba' : generates a probabilistic estimate of each trial.
-
-            'decision_function' : generates a continuous non-probabilistic
-                estimate of each trial.
-        Default: 'predict'
```

Comment:
ok, but let's find an API for chosing between probability and discrete output, maybe based on the scoring.